### PR TITLE
EG 2561: various updates to CENM docs

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**

### CODE REVIEW

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

File: content/en/docs/cenm/1.3/ejbca-plugin.md [hunk 1]
```diff
@@ -112,8 +112,7 @@ administration UI.
 for which certificate will be created. Certificate request is done and after that we collect generated certifciate’s
 chain since node will only accept chains which root certificate matches the one provided in network root truststore.
 
-`submitCRL()` method takes current CRL in form of `crl` argument and set of Certificate Revocation Requests under
-`newCRRs` argument. First of all, revocation of all new revocation requests is performed. After that CRL is updated and
+`submitCRL()` method takes current CRL in form of `crl` argument and a Certificate Revocation List to be updated. First of all, revocation of all new revocation requests is performed. After that CRL is updated and
```
Comment (rnicoll) [inline content/en/docs/cenm/1.3/ejbca-plugin.md:115]: I think weirdly it actually takes in requests and builds the CRL from it. @tubibuto can you confirm please?

**Review by rnicoll — COMMENTED** [review]
One point to verify, but mostly looks good to me.

**Review by szymonsztuka — APPROVED** [review]

File: content/en/docs/cenm/1.3/ejbca-plugin.md (hunk 1 above)
Comment (tubibuto) [reply content/en/docs/cenm/1.3/ejbca-plugin.md:115]: I takes old CRL and new CRRs and builds new CRL from them

**Review by tubibuto — APPROVED** [review]

**Pull request merged**